.github/workflows/tests-misc.yml

```diff
@@ -0,0 +1,379 @@
+name: tests misc
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
+  merge_group:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
```

Comment:
Please add a [comment](https://github.com/paritytech/polkadot-sdk/pull/5340/files#diff-ca6ddd9d3cdff8f696fe610915c9cd9316882794d8ed1e1e4dc6be392c0eb3e3R8) for transparency